### PR TITLE
Ionic documentation

### PR DESCRIPTION
**Pull request by SenNeonoveNoci**

### CODE REVIEW

**Comment by SenNeonoveNoci** [comment]
This can be merged as soon as ionic is gonna merge our plugin.

File: _docs/sdk/ionic-android.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+* Replay session recordings in our web player
+* Capture all of the user interactions and find ones you can about in the Events manager
+* Set your own custom analytics event and do complex funnels/queries in the dashboard
+* Collect referrer value and source of installation per visitor
+
+## Reporting issues and changelog
+
+For more information on how to report issues please check our [Smartlook SDK Support section](https://smartlook.github.io/docs/sdk/support/#how-to-submit-an-issue).
+
+[Smartlook Android SDK changelog](https://github.com/smartlook/smartlook-android-sdk) reports all notable improvements, changes and fixes in SDK releases.
```
Comment (Acro) [inline _docs/sdk/ionic-android.md]: Fix the changelog link.

File: _docs/sdk/ionic-android.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+## Supported versions
+
+Currently, we support API 18+ for recording.
```
Comment (Acro) [inline _docs/sdk/ionic-android.md]: Is this applicable for Ionic?

File: _docs/sdk/ionic-android.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
+---
+title: "Ionic Android"
+subtitle: "SDK for screen recording and analytics for Ionic Android applications."
+description: "This SDK offers several options to developers and/or companies."
+---
+
+## What you can do
+
+* Replay session recordings in our web player
+* Capture all of the user interactions and find ones you can about in the Events manager
```
Comment (janzal) [inline _docs/sdk/ionic-android.md]: you can about?

Comment (SenNeonoveNoci) [reply _docs/sdk/ionic-android.md]: Fixed.

File: _docs/sdk/ionic-android.md (hunk 2 above)
Comment (SenNeonoveNoci) [reply _docs/sdk/ionic-android.md]: Yup

**Pull request merged**